rest-new/pom.xml

```diff
@@ -56,6 +56,16 @@
         </dependency>
 
         <!-- REST dependencies -->
+        <dependency>
+            <groupId>org.jboss.pnc</groupId>
+            <artifactId>notifications</artifactId>
+            <scope>provided</scope>
+        </dependency>
```

Comment:
This should be among the Project dependencies

Reply:
OK

Reply:
It doesn't seem to work without it. See Jenkins job: 4154

Reply:
I just meant move this dependency 2 lines up :)